pkgs/by-name/ey/eyewitness/package.nix

```
@@ -0,0 +1,62 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python311Packages
+, cmake
+, xvfb-run
+, firefox-esr
+, geckodriver
+}:
+
+stdenv.mkDerivation rec {
+  pname = "eye-witness";
+  version = "20230525.1";
+
+  src = fetchFromGitHub {
+    owner = "redsiege";
+    repo = "EyeWitness";
+    rev = "v${version}";
+    hash = "sha256-nSPpPbwqagc5EadQ4AHgLhjQ0kDjmbdcwE/PL5FDL4I=";
+  };
+
+  buildsystems = with python311Packages; [ setuptools ];
+
+  dependencies = with python311Packages; [
```

```suggestion
  buildsystems = with python3Packages; [ setuptools ];

  dependencies = with python3Packages; [
```
Unless absolutely required, python3 should be used